src/main/java/Event.java

```diff
@@ -0,0 +1,43 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    String at;
+    String parsedAt;
+
+    /**
+     * Creates event.
+     * @param description contains description of event.
+     */
+    public Event(String[] description) {
+        super(description[0]);
+        at = (description[1].split(" ", 2))[1];
+        modifyDateFormat();
+    }
+
+    /**
+     * Changes format of Date.
+     */
+    public void modifyDateFormat() {
+        DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HHmm");
+        LocalDateTime parsedDate = LocalDateTime.parse(this.at, inputFormatter);
+        DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy HHmm");
+        this.parsedAt = parsedDate.format(outputFormatter);
+    }
+
+    public Event(String description, String atDate, boolean isDone) {
```

Comment:
Hi! This class is generally well written. You can consider writing javadoc for this method too so its easier for you to revisit it next time!